src/Authentication/Controllers/SystemRegisterController.cs

```diff
@@ -52,9 +52,9 @@ public async Task<ActionResult> GetListOfRegisteredSystems(CancellationToken can
     /// <param name="cancellationToken">The cancellation token</param>
     /// <returns></returns>
     [HttpGet("product/{productId}")]
-    public async Task<ActionResult> GetDefaultRightsForRegisteredSystem(string productId, CancellationToken cancellationToken = default)
+    public async Task<ActionResult> GetRightsForRegisteredSystem(string productId, CancellationToken cancellationToken = default)
     {
-        List<DefaultRight> lista = await _systemRegisterService.GetDefaultRightsForRegisteredSystem(productId, cancellationToken);
+        List<Right> lista = await _systemRegisterService.GetRightsForRegisteredSystem(productId, cancellationToken);
```

Comment:
We should use one naming for this. Now we use productId somewhere and customSystemID other places

Reply:
yes, have updated now